server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_4__add_odc_user_recent_browsing.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `odc_user_recent_browsing` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
+  `project_id` bigint NOT NULL COMMENT 'project id, FK refer to collaboration_project.id',
+  `user_id` bigint NOT NULL COMMENT 'user id，FK refer to iam_user.id',
+  `item_type` varchar(32) NOT NULL COMMENT 'browsing item type, contains: WROKSHEET,GIT_REPO',
+  `item_id` bigint NOT NULL COMMENT 'browsing item id',
+  `browse_time` datetime  NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'browsing time',
```

Comment:
column naming suggestion:
resource_type, resource_id, last_access_time

Reply:
Okay，I'll make this change.